keras_cv/layers/object_detection/object_detection_label_encoder.py

```diff
@@ -16,10 +16,9 @@
 from tensorflow.keras import layers
 
 from keras_cv import bounding_box
-from keras_cv.models.object_detection.retina_net.__internal__ import utils
 
 
-class LabelEncoder(layers.Layer):
+class ObjectDetectionLabelEncoder(layers.Layer):
```

Comment:
Nice!
On a side note, maybe RetinaNetLabelEncoder? (As this won't be used in FasterRCNN)